README.md

```diff
@@ -9,10 +9,11 @@ The repository is structured as follows:
 - [Quantum machine learning and optimization with PennyLane](#pennylane)
 - [Amazon Braket features](#braket)
 - [Amazon Braket Hybrid Jobs](#jobs)
-- [Creating a Conda environment](#conda)
+- [Pulse Control](#pulse)
+- [Qiskit with Braket](#qiskit)
 
 ---
-## <a name="simple">Simple circuits and algorithms</a>
+## <a name="simple">I'm new to quantum</a>
```

Comment:
Should the title of the ToC change here?

Reply:
Done. Not sure about the naming, still. Waiting for Jordan to weigh in.